opensir/models/model.py

```diff
@@ -175,10 +181,34 @@ def fit(self, t_obs, n_i_obs, population, fit_index=None):
             Model: Reference to self
         """
 
+        # If input not np.ndarray, cast
+        if not isinstance(t_obs, np.ndarray):
+            t_obs = np.array(t_obs)
+
+        if not isinstance(n_i_obs, np.ndarray):
+            n_i_obs = np.array(n_i_obs)
+
+        # Raise error if any time or number of infected is negative
+        if any(t_obs < 0) | any(n_i_obs < 0):
+            raise self.InvalidInputsForFittingError(Exception)
+
+        # Raise error if any time or number of infected lists
+        # is non monotonically increasing
+        if any(np.diff(t_obs) < 0) | any(np.diff(n_i_obs) < 0):
```

Comment:
is this true for infected? Does this number reflect the number of "Active cases" (I(t)) or total infected cases ( int( I (t), 0, T) ) ??

Reply:
You are right! this is only mandatory for t_obs. Good spot! I was thinking on C(t) which is the number of cases reported. All of this will be clarified